azure-pipelines.yml

```diff
@@ -88,7 +94,7 @@ jobs:
     displayName: Build Android [Legacy Renderers]
     vmImage: $(macOSVmImage)
     targetFolder: Xamarin.Forms.ControlGallery.Android/legacyRenderers/
-    androidProjectArguments: '/t:"Rebuild;SignAndroidPackage"'
+    androidProjectArguments: '/t:"Rebuild;SignAndroidPackage" /bl'
```

Comment:
we usually append a target filename to `/bl` so subsequent builds won't overwrite the same binlog, such as `/bl:$(Build.ArtifactStagingDirectory)\legacy.binlog`